pkcs11-driver.opam

```diff
@@ -18,7 +18,7 @@ depends: [
   "dune" {>= "2.0.0"}
   "ocaml" {>= "4.07.0"}
   "ounit" {with-test}
-  "pkcs11" {>= "0.18.0"}
+  "pkcs11" {>= "1.0.2"}
```

Comment:
Is there really an incompatibility with versions between 0.18.0 and 1.0.1? Version constraints are not a bad idea but I'm wondering if we really broke the API.

Reply:
Well, if you want to install pkcs11-driver 1.0.2 on macos, you need pkcs11 1.0.2. That's why I changed it, but maybe I don't understand how opam works and it's just done automatically here ?

Reply:
Well, opam guys decided not to change the version in the repository so I think we should do the same ?

Reply:
I decided not to make a new version because that's a typical thing to do in the opam repository (the version constraints were wrong so we fixed them). Of course if we had made any change to the code they would have required us to release it with a new version.

> Well, if you want to install pkcs11-driver 1.0.2 on macos, you need pkcs11 1.0.2. That's why I changed it

Why would you need it? If pkcs11 0.18.0 and 1.0.1 have `available: [os != macos]` then you don't need to do anything as your macOS won't even try to install them.